src/minivhd_util.c

```diff
@@ -227,4 +232,41 @@ const char* mvhd_strerr(MVHDError err) {
     default:
         return "unknown error";
     }
+}
+
+int64_t mvhd_ftello64(FILE* stream)
+{
+#ifdef _MSC_VER
+    return _ftelli64(stream);
+#else
+    return ftello64(stream);
+#endif
+}
+
+int mvhd_fseeko64(FILE* stream, int64_t offset, int origin)
+{
+#ifdef _MSC_VER
+    return _fseeki64(stream, offset, origin);
+#else
+    return fseeko64(stream, offset, origin);
+#endif
+}
```

Comment:
Maybe I should do some BSD checks here as well, as that may not do fseeko64 either.

Large file support is messy 😢

Reply:
Yeah, adding support for more OSs/compilers is a good idea. I think the vast majority of use of MiniVHD would be covered by what is here already.